sdk/core/azure-core/src/main/java/com/azure/core/util/MetricsOptions.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+import com.azure.core.http.ProxyOptions;
+
+/**
+ * Metrics configuration options for clients.
+ */
+public class MetricsOptions {
```

Comment:
`TracingOptions` - totally, along with custom TracerProvider support - currently there is no way to even pass tracing options as Tracer is the SPI and can't have parameters :( 

So adding it would not be so straightforward, but I'll get there.